chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -106,6 +110,10 @@
       done
     EOF
   end
+  # Deletes postgres data dir in non-HA scenario
+  directory "/var/lib/pgsql/data" do
+    action :delete
```

Comment:
> @vuntz I am a somewhat undecided on this, could you please be the tie-breaker ;)

I'm not convinced we would really hit any issue with disk space (do we have any figure that would indicate that?), and it feels safer to keep an old copy in case there's some broken user manipulation. At least, that's what I would do if I were to do things manually ;-)